platform/vs/docker-sonic-vs.mk

```diff
@@ -2,9 +2,7 @@
 
 DOCKER_SONIC_VS = docker-sonic-vs.gz
 $(DOCKER_SONIC_VS)_PATH = $(PLATFORM_PATH)/docker-sonic-vs
-$(DOCKER_SONIC_VS)_DEPENDS += $(SWSS) \
```

Comment:
Curious why `$(SWSS)` is removed here? Is it because it's now taken care of in the addition on line 46?

Reply:
Yes, `docker-sonic-vs` will be based on top of `docker-swss-layer-bullseye`, which will contain SWSS.

The initial intention of that layer was to save space by having SWSS be in a single layer and then other containers can be built on top of this layer. In this case, since `docker-sonic-vs` won't actually be part of the SONiC image, there's effectively no impact.